jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -188,28 +187,27 @@ pipeline {
 
                     stash includes: "**", name: "validate-artifacts-$BUILD_NUMBER"
 
-                    for (architecture in architectureList) {
-                        validateDistributions = [:]
-                        for (distribution in distributionList) {
+                    def validateDistributions = [:]
+
+                    // Create parallel stages for all combinations of distribution and architecture
+                    for (distribution in distributionList) {
+                        for (architecture in architectureList) {
                             def local_architecture = architecture.trim()
                             def local_distribution = distribution.trim()
                             def local_platform = mapDistributionToPlatform.getOrDefault(local_distribution, "linux")
-                            def local_distribution_index = distributionList.indexOf(local_distribution)
-                            def wait_seconds = local_distribution_index * 10
-
-                            echo "Validating Distribution: ${local_distribution}-${local_architecture}"
 
+                            // Skip zip-arm64 combination
                             if (local_distribution == 'zip' && local_architecture == 'arm64') {
```

Comment:
Hi @gaiksaya can you also add a condition to exclude one docker run ?. We are currently running twice for x64 and arm64.

Reply:
We need to validate both x64 and arm64 docker layers. Keeping it as is.
Thanks!